tf/scripts/backend_user_data.sh

```
@@ -1,20 +1,86 @@
-# Construir la imagen Docker para el backend
+# Build the Docker image
 cd /home/ec2-user/backend
-sudo docker build -t lti-backend .
+docker build -t lti-backend .
+
+# Run the backend container with Datadog env vars
+docker run -d \
+  --name lti-backend \
+  --restart unless-stopped \
+  -p 8080:8080 \
+  -e DATABASE_URL="postgresql://${db_user}:${db_password}@localhost:5432/${db_name}" \
```

Shellcheck (0.11.0)

[warning] db_user is referenced but not assigned.

(SC2154)